draft-holmberg-mmusic-t140-usage-data-channel.xml

```diff
@@ -387,6 +387,10 @@ Answer:
             If the gateway detects that the T.140 data channel has failed and got torn down, once the data channel has been reestablished
             the gateway SHOULD insert the T.140 missing text marker <xref target="T140ad1"/> in the data sent on the outgoing RTP stream.
```

Comment:
Add last: "if it detects or suspects that transmitted T140 data was lost."

Reply:
It is mentioned in the bullet above, but I guess we can mention it explicitly also for the data channel failure case.